geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/DescribeMappingCommand.java

```diff
@@ -145,7 +145,7 @@ private ResultModel buildResultModel(ArrayList<DescribeMappingResult> describeMa
         regionConfig,
         group)) {
       results.add(buildDescribeMappingResult(mapping, regionConfig.getName(),
-          MappingCommandUtils.isMappingAsync(cacheConfig, regionConfig), group));
+          MappingCommandUtils.isMappingSynchronous(cacheConfig, regionConfig), group));
```

Comment:
It looks like you found a bug in isMappingAsync and fixed it by changing the name.
Does this indicate that we are missing a test on DescribeMappingCommand that tests this boolean result?